### DISCUSSION TITLE
Following hyperlinks to other orgfiles

### DISCUSSION THREAD

**Discussion by seflue** [nvim-orgmode/orgmode#580] [Q&A] [answered]
Currently I am struggeling to follow hyperlinks. I expected "gd" to work, but it doesn't and I wasn't able to find a hint if and how this is mapped. Because of that I am not sure, if I shadowed "gd" with some other configuration (e.g. treesitter mappings) or if this feature doesn't exist or if I have to configure it. Thanks in advance for any advice.

**Comment by kristijanhusak** [comment, marked as answer]
Mapping is `<leader>oo`

**Reply by seflue** [reply]
Many thanks! Would be great, if the tag `orgmode-org_open_at_point`  could be mentioned in the `HYPERLINKS` section in the documentation. Without case-insensitive fulltext search it is hard to find this in the documentation.

**Reply by jgollenz** [reply]
@seflue adding missing documentation is a great way to get started in open source and helps us out a lot :slightly_smiling_face:  Feel free to open a PR :+1:

**Reply by seflue** [reply]
@jgollenz Sure. I opened a PR, see #582.
